admin/code/LeftAndMain.php

```diff
@@ -355,10 +356,12 @@ protected function getSchemaResponse($form) {
 	 * based on the request data.
 	 *
 	 * @param Form $form
+	 * @param String $id Optional, will default to the current request URL
 	 * @return array
 	 */
-	protected function getSchemaForForm(Form $form) {
+	protected function getSchemaForForm(Form $form, $id = null) {
 		$request = $this->getRequest();
+		$id = $id ? $id : $request->getURL();
```

Comment:
I like the idea of `Form->Link()`, it's a nice middle ground. Looking at the FormFactory work that was just merged yesterday, this may not work out when Forms are context agnostic.

`schema.action` almost looks to be the same as `Form->Link()`, maybe I'm missing a distinction between the two.

We can pull this discussion/brainstorming out to a separate issue, I agree that it doesn't need to block merge!